scipy/interpolate/_bsplines.py

```diff
@@ -744,6 +744,9 @@ def make_interp_spline(x, y, k=3, t=None, bc_type=None, axis=0,
         except TypeError:
             raise ValueError("Unknown boundary condition: %s" % bc_type)
 
+    # BSpline requires axis>=0
+    axis = axis % y.ndim
```

Comment:
The "standard" way to write this is 
```
if axis < 0:
    axis += y.ndim
    if axis < 0:
        raise ValueError("axis {} is out of bounds".format(axis - y.ndim))
```

Reply:
Sure. Was just moving one line of code originally there upward.